.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Please remove this.
I have .gitignore locally, and it conflicts.
Or add
```
.gitignore
node_modules/
```
to your forked local .gitignore.

Reply:
Can you commit your `.gitignore`?

Reply:
It includes
```
.gitignore
```
so it can not be uploaded.

Reply:
No problems, lets play soccer.

Reply:
The reason why I don't put the .gitignore in the master branch is that it would prevent who forked this repo from choosing which files to expose and which not to do so.